src/detectors/BEMC/BEMC.cc

```diff
@@ -28,13 +30,39 @@ void InitPlugin(JApplication* app) {
 
   InitJANAPlugin(app);
 
+  // Make sure left and right use the same value
+  decltype(SimCalorimeterHitProcessorConfig::attPars) EcalBarrelScFi_attPars = {0.416212, 747.39875,
```

Comment:
Some of the attenuation parameters must be dimensional of some sort. Please add dimensions here. From the formula it seems that `[0]` is dimensionless, but `[1]` and `[2]` are `1/dd4hep::mm` presumably.